frontend/app/routes/create.$type.tsx

```diff
@@ -97,6 +100,8 @@ export default function Create() {
       )}
       <ScriptModal
         title="Your script"
+        tooltip="Copy your script, place it before the closing body tag in your website, or place it into a script type element in instances when you have a site management software (ex: wordpress, etc). 
```

Comment:
```suggestion
        tooltip="Copy your script and insert it just before the closing </body> tag on your website, or place it within a <script> tag if you're using site management software (e.g., WordPress, etc.). 
```

Reply:
I partially added your suggestion, because in many cases users have specific elements they need to use from what's available for them (see screenshot of one of the WP possibilities), and our script is already wrapped in a <script> tag

![Screenshot from 2024-11-14 23-32-42](https://github.com/user-attachments/assets/2b7895f4-c1a2-4d98-ba5e-0833088ed5af)